client/src/App.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const App = () => {
+  <div>
+    <h1>Hello World</h1>
+  </div>;
+};
```

Comment:
nit: can you add a blank line in the end of files to avoid the "no new line at end of file" github alert?